src/libraries/System.Text.Json/tests/System.Text.Json.Tests/SourceGenRegressionTests/Net90/Net90GeneratedContext.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Source files represent a source generated JsonSerializerContext as produced by the .NET 8 SDK.
+// Used to validate correctness of contexts generated by previous SDKs against the current System.Text.Json runtime components.
+// Unless absolutely necessary DO NOT MODIFY any of these files -- it would invalidate the purpose of the regression tests.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace System.Text.Json.Tests.SourceGenRegressionTests.Net90
+{
+    //[JsonSerializable(typeof(WeatherForecastWithPOCOs))]
+    //[JsonSerializable(typeof(ClassWithCustomConverter))]
+    //[JsonSerializable(typeof(MyLinkedList))]
```

Comment:
What are these?

Reply:
The attributes that would have otherwise prompted the generator to produce the code that is being committed.